images/gcb-docker-gcloud/Dockerfile

```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-ARG GO_VERSION=1.22.9
+ARG GO_VERSION=1.23.1
```

Comment:
why not 1.23.4?

Reply:
Misread https://go.dev/doc/devel/release .. will update

Reply:
Thx for catching this, done